### PR TITLE
Analytic Pandunia 2021

### PR DESCRIPTION
**Pull request by barumau**
This is an experimental branch for trying out an analytic grammar for Pandunia as proposed in the Reddit thread called "[No more word class markers?](https://www.reddit.com/r/pandunia/comments/ng2tys/no_more_word_class_markers/)".

I don't know yet how far this will go but this is _something that I need to get out of my system_.

### CODE REVIEW

File: engli/102_ABC.md [hunk 1]
```diff
@@ -65,57 +46,78 @@ which are pronounced as the consecutive vowels with or without a hiatus in betwe
...
+| H        | h (x)  | |
+| J        | dʒ     | Always like J in judge_ or the soft _g_ in _gel_. |
+| K        | k      | |
+| L        | l      | |
+| M        | m      | |
+| N        | n (ŋ)  | |
+| P        | p      | |
+| R        | r (ɹ)  | Preferably trilled as in Scottish and Indian English, or rolled as in American English. Never silent! |
+| S        | s      | Always voiceless like _s_ in _sissy_. |
+| Sh       | ʃ      | Like _sh_ in _shop_. |
```
Comment (ChristianSi) [inline engli/102_ABC.md:81]: Why do you want to change the consonant spellings to "ch" and "sh"? That seems unrelated to analytic Pandunia.

File: engli/106_modologe.md [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
+The adverb and adjective can together modify a verb or a phrase.
 
-**dayo sundari jan**
-– very beautiful person  
-**lilo sundari jan**
-– somewhat beautiful person
+**mo _tre bon_ sona.**
+– I very well sleep.  
+**mo sona _tre bon_.**
+– I sleep _very well_.
```
Comment (ChristianSi) [inline engli/106_modologe.md:38]: But cannot one say: "mo sona bon" – 'I sleep well'? I don't think all adverbs will include a degree word?

File: engli/108_visloge.md [hunk 3]
```diff
@@ -6,26 +6,24 @@ Pronouns can substitute nouns and noun phrases.
 
 | Singular    | Plural       |
 |:------------|:-------------|
-| **me**      | **mome**     |
+| **mo**      | **momen**    |
```
Comment (ChristianSi) [inline engli/108_visloge.md:9]: -men works in the 2nd and 3rd person, but not so well in the 1st – "we" isn't the plural of "I".

File: engli/110_jumle.md [hunk 4]
```diff
@@ -7,74 +7,72 @@ They may consist of only one word, which draws the listener's attention.
...
 – I am good.  
-**le novi.**
+**le nove.**
 – It is new.  
 **le Sara.**
-– She is Sara.  
-**Sara boni.**
+– She is Sara.
+
+When the subject is a noun, the connecting verb **si** (_to be_) is necessary.
```
Comment (ChristianSi) [inline engli/110_jumle.md:27]: Why? And Chinese and many creole languages, one doesn't use a copula in such cases.

**Comment by ChristianSi** [comment]
I've made some inline comments. Additionally I wonder whether this would not be ambiguous (or at least hard to read) in complex sentences. For example: "me vol abla gova dom" – how does one know where the object begin? And if the subjects weren't a noun, how would one know where the verb starts? If words can be used as nouns, adjectives and verbs, couldn't this sentence also mean something like: "I want an able going to the house"? How du I know that "abla gova" is still part of the verb?

For an analytical grammar to work in an easy manner, I think you would need some kind of particles marking where noun or verb phrases begin. For example, in Elefen most noun phrases begin with one of the articles "la" or "un" or another determiner. Without articles, Elefen's analytical grammar wouldn't work.

Another option would be to mark verbs, say "me (pres) vol (to) abla (to) gova dom", where (pres) needs to be substituted with a present-tense particle and (to) with an infinitive particle. Some kind of particle marking adverbs would likely be needed as well.

Related: how to turn adjectives into adverbs modifying other adjectives – e.g. how to translate "I find him/her amazingly intelligent?" If one doesn't want to use an affix, this seems to require a marker particle too.

File: engli/103_logotipe.md [hunk 5]
```diff
@@ -1,180 +1,64 @@
...
+Content words are like bricks of information and structural words are like the mortar that holds them together.
+
+In Pandunia, it is easy to identify structure words because they always consist of only one syllable.
+Content words, on the other hand, are typically longer.
+Structure words are best explained in the grammar,
+whereas content words are translated in the dictionary.
+
+## Word classes
+
+Content words can be categorized into word classes as follows:
```
[the remark this answers is not in the document]
Comment (barumau) [reply engli/103_logotipe.md:38]: Right. Thanks!

File: engli/106_modologe.md (hunk 2 above)
Comment (barumau) [reply engli/106_modologe.md:38]: In practice, yes, one could say it like that too. I prefer to use a degree adverb together with adjectives because degree adverbs are structure words that help to identify the roles of content words. A series of content words can be interpreted in many different ways, as I tried to demonstrate in our debate in Reddit, though I must admit that this specific sentence is not likely to be misunderstood.

File: engli/107_nume.md [hunk 6]
```diff
@@ -7,48 +7,48 @@ They are put before the word or phrase that they qualify.
...
-- 6 **sisi**
-- 7 **ceti**
-- 8 **bati**
-- 9 **novi**
+- 2 **du**
+- 3 **tri**
+- 4 **si**
+- 5 **go**
+- 6 **xe**
+- 7 **ci**
```
[the remark this answers is not in the document]
Comment (barumau) [reply engli/107_nume.md:35]: You're right. I will try something else.

File: engli/108_visloge.md (hunk 3 above)
Comment (barumau) [reply engli/108_visloge.md:9]: I don't see any real problem in there. In a strict sense you are right but pronouns are seldom used so strictly. I/we means the person/people I speak for, myself or my people. You/you all means the person/people I speak to, yourself or your people. He/she/they means the person/people I speak about.

It's possible extend plurals to people that are not really involved by saying things like "We the Finns admire you Germans" even if in reality there is only one Finn saying it and only one German hearing it. So the plural pronouns can also represent others than only the speaker the the listener(s).

File: engli/110_jumle.md (hunk 4 above)
Comment (barumau) [reply engli/110_jumle.md:27]: This is one of those structures where the word class markers did really well. There is need to structurally differentiate things like "crystal-clear" and "crystal is clear".

What Chinese does is that it inserts a degree adverb between the subject and the adjective: wǒ māma hěn hǎo. Pandunia could use that structure: mo de mama tre bon. Sara tre bon. kristal tre klar. idea tre kristal klar.

**Comment by barumau** [comment]
Thanks for your comments here and in Telegram and Reddit. I will merge the changes to the main branch now. Let's solve remaining issues there later.
